app/controllers/email/CaptureEmailController.scala

```diff
@@ -100,4 +101,76 @@ class CaptureEmailController @Inject()(val vatSubscriptionService: VatSubscripti
       case (None, _) => Future.successful(errorHandler.showInternalServerError)
     }
   }
+
+  def showEmail: Action[AnyContent] = contactPreferencePredicate.async { implicit user =>
```

Comment:
Can this action and the submit version of it be renamed to differentiate it from the other actions/journey? Both sets of actions involve changing email so this is not currently clear. `showPrefJourney`, perhaps?